python/foglamp/services/south/server.py

```diff
@@ -187,6 +187,7 @@ def run(self):
                 self._plugin.plugin_shutdown(self._plugin_handle)
             except Exception as ex:
                 _LOGGER.exception("Unable to stop plugin '{}'".format(self._name))
+                raise ex
```

Comment:
This has been deliberately removed as it allows a plugin to prevent a south plugin from shutting down.